src/theme.js

```diff
@@ -0,0 +1,112 @@
+import theme from "@rebass/preset"
+
+const customTheme = {
+  ...theme,
+  breakpoints: [320, 768, 1024, 1440],
+  colors: {
+    darkBlue: "#0C1D60",
+    lightBlue: "#7BD5F5",
+    gold: "#F7D663",
+    white: "#FFF",
+  },
+  fonts: {
```

Comment:
Not sure what the syntax for declaring a single font style is, but can you make it just `Avenir` with these fallbacks: `Helvetica, Arial, sans-serif`?